src/main/java/nextstep/subway/section/domain/Section.java

```diff
@@ -0,0 +1,116 @@
+package nextstep.subway.section.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import nextstep.subway.line.domain.Line;
+import nextstep.subway.station.domain.Station;
+
+@Entity
+public class Section {
+
+    private static final String NEW_DISTANCE_BIGGER_ERROR = "새로운 구간의 길이가 기존의 길이보다 길수 없습니다.";
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "up_station_id", nullable = false)
+    private Station upStation;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "down_station_id", nullable = false)
+    private Station downStation;
+
+    @Embedded
+    private Distance distance;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "line_id", nullable = false)
+    private Line line;
```

Comment:
구간에 노선과 상행역, 하행역은 필수라 optional = false 를 명시적으로 선언해줘도 좋을 것 같아요
```suggestion
    @ManyToOne(optional = false, fetch = FetchType.LAZY)
    @JoinColumn(name = "up_station_id", nullable = false)
    private Station upStation;

    @ManyToOne(optional = false, fetch = FetchType.LAZY)
    @JoinColumn(name = "down_station_id", nullable = false)
    private Station downStation;

    @Embedded
    private Distance distance;

    @ManyToOne(optional = false, fetch = FetchType.LAZY)
    @JoinColumn(name = "line_id", nullable = false)
    private Line line;
```